src/Me/Navigation/Navbar.js

```diff
@@ -20,8 +23,13 @@ const Navbar = () => {
           src={`${process.env.PUBLIC_URL}/codingcoach-logo-192.png`}
           alt="Logo"
         />
-        <MenuItem to="/me/home" icon={IconHome} label="Home" />
+        <MenuItem to="/me" icon={IconHome} label="Home" />
+        <MenuItem to="/me/requests" icon={IconHome} label="Req" />
```

Comment:
Is there such an item? The requests list card should be under home anyway.

Reply:
It's just a placeholder. I wasn't sure where to put it. 👍🏼